src/Components/Blazor/Build/src/targets/All.targets

```diff
@@ -13,6 +13,9 @@
 
     <!-- The Blazor build code can only find your referenced assemblies if they are in the output directory -->
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+
+    <!-- By default, enable debugging for debug builds. -->
+    <BlazorEnableDebugging Condition="'$(Configuration)' == 'Debug' AND '$(BlazorEnableDebugging)' == ''">true</BlazorEnableDebugging>
```

Comment:
Prop files from NuGet packages are imported prior to `Configuration` being initialized. This moves the assignment to a later point.